client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -57,6 +57,9 @@ const onboarding: Flow = {
 			[]
 		);
 
+		// we are only interested in the initial goals value when the user enters the goals step
```

Comment:
@vykes-mac  Should this comment say "when the user leaves the goals step" ?

Reply:
Oh oh, nevermind.  Of course when `calypso_signup_start` fires !!  "Start" being the key word! 😄

Reply:
This is more from the perspective of the user already had so goals and refresh the page on the goals step.